docs/api/cases/cases-api-create.asciidoc

```diff
@@ -0,0 +1,203 @@
+* `destIp`: A comma separated list of destination IPs.
+* `malwareHash`: A comma separated list of malware hashes.
+* `malwareUrl`: A comma separated list of malware URLs.
+* `sourceIp`: A comma separated list of source IPs.
+////
+* `priority`: The priority of the incident.
+* `category`: The category of the incident.
+* `subcategory`: The subcategory of the incident.
+
+For {jira} connectors, refer to <<jira-action-configuration>>. For example:
+
+* `issueType`: The type of the issue.
+* `priority`: The priority of the issue.
+* `parent`: The key of the parent issue (Valid when the issue type is `Sub-task`).
+
+For {ibm-r} connectors, refer to <<resilient-action-configuration>>. For example:
+
+* `issueTypes`: The type of the incident.
+* `severityCode`: The severity code of the incident.
+
+For {swimlane} connectors, refer to <<swimlane-action-configuration>>. For
+example:
+
+* `caseId`: The case ID.
+//TBD: Is this correct or should it be comments and severity?
+--
+
+`id`::
+(Required, string) The identifier for the connector. To create a case without a
+connector, use `none`.
+//To retrieve connector IDs, use <<cases-api-find-connectors>>).
+
+`name`::
+(Required, string) The name of the connector. To create a case without a
+connector, use `none`.
+
+`type`::
+(Required, string) The type of the connector. Valid values are: `.jira`, `.none`,
+`.resilient`,`.servicenow`, `.servicenow-sir`, and `.swimlane`. To create a case
+without a connector, use `.none`.
+====
+
+`description`::
+(Required, string) The description for the case.
+
+`owner`::
+(Required, string) The application that owns the case. Valid values are:
+`cases`, `observability`, or `securitySolution`. This value affects
+whether the case is visible in the {stack-manage-app}, {observability}, or
+{security-app}.
+
+`settings`::
+(Required, object)
+An object that contains the case settings.
++
+.Properties of `settings`
+[%collapsible%open]
+====
+`syncAlerts`:: 
+(Required, boolean) Turns alert syncing on or off.
+====
+
+`tags`::
+(Required, string array) The words and phrases that help
+categorize cases. It can be an empty array.
+
+`title`::
+(Required, string) A title for the case.
+
+=== Response code
+
+`200`::
+   Indicates a successful call.
+
+=== Example
+
+[source,sh]
+--------------------------------------------------
+POST api/cases
+{
+  "description": "James Bond clicked on a highly suspicious email
+  banner advertising cheap holidays for underpaid civil servants.",
+  "title": "This case will self-destruct in 5 seconds",
+  "tags": [
+    "phishing",
+    "social engineering"
+  ],
+  "connector": {
+    "id": "131d4448-abe0-4789-939d-8ef60680b498",
+    "name": "My connector",
+    "type": ".jira",
+    "fields": {
+      "issueType": "10006",
+      "priority": "High",
+      "parent": null
+    }
+  },
+  "settings": {
+    "syncAlerts": true
+  },
+  "owner": "securitySolution"
+}
+--------------------------------------------------
+// KIBANA
+
```

Comment:
I removed the example of how to create a case with no connector, since I figured the more complicated example was more useful. However, I can add it back if needed.